logprep/framework/rule_tree/node.py

```diff
@@ -9,7 +9,13 @@
 class Node:
```

Comment:
We could do that, but it would require every FilterExpression to have a children attribute.
FilterExpression like the StringFilterExpression can never have children.